src/core/credentials/transport/google_default/google_default_credentials.cc

```diff
@@ -410,6 +411,37 @@ grpc_channel_credentials* grpc_google_default_credentials_create(
   return result;
 }
 
+grpc_channel_credentials*
+grpc_google_default_credentials_create_with_dual_call_credentials(
```

Comment:
We don't need this New API. Instead we can modify the existing `grpc_google_default_credentials_create` by adding a second argument, which can default to nullptr.